adapters/oidc/spring-boot/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfiguration.java

```diff
@@ -209,7 +209,8 @@ public void customize(Server server) {
                 }
             }
 
-            WebAppContext webAppContext = server.getBean(WebAppContext.class);
+            WebAppContext webAppContext = (WebAppContext) server.getHandler();;
```

Comment:
One can register multiple servlets in a Spring Boot application by using a [`ServletRegistrationBean`](http://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/web/servlet/ServletRegistrationBean.html), which would create multiple contexts in the application.

This should however not block this PR, it is something to note, maybe in the docs or something?